code/game/objects/structures/ai_core.dm

```diff
@@ -197,7 +245,16 @@
 				if(istype(P, /obj/item/mmi) && !brain)
 					var/obj/item/mmi/M = P
 					if(!M.brain_check(user))
-						return
+						var/install = tgui_alert(user, "This brain is inactive, would you like to make an inactive AI?", "Installing AI Brain", list("Yes", "No"))
+						if(install == "No")
+							return
+						else
+							if(!user.transferItemToLoc(M, src))
+								return
+							brain = M
+							balloon_alert(user, "added [M.name] to frame")
```

Comment:
```suggestion
							balloon_alert(user, "added mmi to frame")
```

We don't want to use custom names in balloon alerts to guarantee they are short.